sensor_calibration_manager/sensor_calibration_manager/calibrators/drs/mapping_based_lidar_lidar_calibrator.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Tier IV, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict
+
+import numpy as np
+
+from sensor_calibration_manager.calibrator_base import CalibratorBase
+from sensor_calibration_manager.calibrator_registry import CalibratorRegistry
+from sensor_calibration_manager.ros_interface import RosInterface
+from sensor_calibration_manager.types import FramePair
+
+
+@CalibratorRegistry.register_calibrator(
+    project_name="drs", calibrator_name="mapping_based_lidar_lidar_calibrator"
```

Comment:
@knzo25 
I appreciate your updating the calibration tool. I wonder if here perhaps needs to align the file name under `sensor_calibration_manager/launch/drs` (?)
```suggestion
    project_name="drs", calibrator_name="mapping_based_base_lidar_calibrator"
```

Reply:
Sorry, I did not understand this issue. Let me ask you in the office later